utils/build.ps1

```diff
@@ -2756,6 +2771,11 @@ if (-not $SkipBuild) {
 
   Invoke-BuildStep Build-CMark $HostArch
   Invoke-BuildStep Build-Compilers $HostArch
+
+  $InstallTo = "$($HostArch.ToolchainInstallRoot)\usr"
```

Comment:
I would just inline the single use of this variable.

Reply:
We will have a second `foreach` once we got Android to work